frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java

```diff
@@ -41,6 +47,19 @@ public GitMachetePanel(Project project) {
     setToolbar(createGitMacheteVerticalToolbar(graphTable).getComponent());
     add(createShrinkingWrapper(selectionComponent), BorderLayout.NORTH);
     setContent(ScrollPaneFactory.createScrollPane(graphTable));
+
+    addAncestorListener(new AncestorListenerAdapter() {
+      @Override
+      public void ancestorAdded(AncestorEvent event) {
```

Comment:
Maybe drop a comment about what's the specific event that we aim to capture here

Reply:
done, here is an example event

![image](https://user-images.githubusercontent.com/19799111/99282951-a5342300-2834-11eb-96fd-68154e6dc4c4.png)